src/component/menu/ApiDetails/ApiDetails.tsx

```diff
@@ -0,0 +1,106 @@
+import useUiConfig from '../../../hooks/api/getters/useUiConfig/useUiConfig';
+import { ReactElement } from 'react';
+import ConditionallyRender from '../../common/ConditionallyRender';
+import { IVersionInfo } from '../../../interfaces/uiConfig';
+
+interface PartialUiConfig {
+	name: string;
+	version: string;
+	slogan?: string;
+	environment?: string;
+	versionInfo?: IVersionInfo;
+}
+
+interface Props {
```

Comment:
I'm a bit back and forth on this, but we've used ComponentNameProps as a convention for props. I think we could easily just call it Props when it lives with a component, especially since it's not exported anywhere. But I would like us to follow the same convention.